frontend/src/components/auth/signUpForm.js

```diff
@@ -118,10 +112,10 @@ const InputFormContainer = ({ onRegister }) => {
                   <Col>
                     <Box align="right">
                       <Route
-                        render={({ history }) => (
+                        render={() => (
```

Comment:
I am not sure, but probably this render on button is not necessary when we don't use history here anymore

Reply:
Probably. I couldn't test register flow, because of backend issues, so let's leave it for later. Until then, I will rename SignIn -> login, SignUp -> register